Marlin/src/inc/SanityCheck.h

```diff
@@ -1594,113 +1520,52 @@ static_assert(X_MAX_LENGTH >= X_BED_SIZE && Y_MAX_LENGTH >= Y_BED_SIZE,
   #error "select hardware UART for TMC2208 to use both TMC2208 and ENDSTOP_INTERRUPTS_FEATURE."
 #endif
 
-#if ENABLED(HYBRID_THRESHOLD) && DISABLED(STEALTHCHOP)
-  #error "Enable STEALTHCHOP to use HYBRID_THRESHOLD."
-#endif
-
-#if ENABLED(TMC_Z_CALIBRATION) && !Z_IS_TRINAMIC && !Z2_IS_TRINAMIC
-  #error "TMC_Z_CALIBRATION requires at least one TMC driver on Z axis"
```

Comment:
Does this `TMC_Z_CALIBRATION` test get replaced elsewhere, or is `TMC_Z_CALIBRATION` being dropped?